### PR TITLE
Feature/use api client

### LINKED ISSUE #186: Make GisExcerptConverter.extract_excerpts more readable
Estimated duration: ~0.25d

Suggestions:
- [ ] extract the `for result_file_name in os.listdir(settings.RESULT_MEDIA_ROOT):` loop into a new function (e.g. named `collect_output_files(...)`). See Pull request [comment](https://github.com/geometalab/osmaxx/pull/178/files#r38641128).

### LINKED ISSUE #263: Webapp: Integrate Conversion Service with Webapp
Get the conversion-service working with the webapp.

### PR DESCRIPTION
**Pull request by hixi**
Switch Conversion Service.

Closes #263 

Also closes #186
(because it removes the to-be-simplified code)

Reviewed by:
- [ ] @das-g

### CODE REVIEW

**Comment by hixi** [comment]
The added .yaml don't have to be reviewed, they were created automatically by [vcrpy](https://pypi.python.org/pypi/vcrpy).

File: compose-development.yml [hunk 1]
```diff
@@ -32,9 +32,10 @@ webappdev:
     - sourcedev
     - shareddatadev
   links:
-    - rabbitmqdev:rabbit
     - databasedev:database
-    - celerydev:celery
+  external_links:
+    # foldername_api_1 where foldername corresponds to where the osmaxx-postgis-conversion is checked out
```
Comment (das-g) [inline compose-development.yml]: I'm don't think I understand what this comment is saying.

File: osmaxx-py/config/settings/test.py [hunk 2]
```diff
@@ -1,6 +1,15 @@
 from .local import *  # noqa
 
-OSMAXX['download_file_name'] = '%(excerpt_name)s-%(content_type)s-%(id)s.%(file_extension)s'
+DEBUG = False
+
+OSMAXX_TEST_SETTINGS = {
+    'download_file_name': '%(excerpt_name)s-%(content_type)s-%(id)s.%(file_extension)s',
+    'CONVERSION_SERVICE_URL': 'http://localhost:8901/api/',
+    'CONVERSION_SERVICE_USERNAME': 'osmaxxi',
```
Comment (das-g) [inline osmaxx-py/config/settings/test.py]: Not that it matters, but: Why the `'i'` at the end?

File: compose-common.yml [hunk 3]
```diff
@@ -20,7 +20,6 @@ base:
     - DJANGO_MEDIA_ROOT=/data/media
     - DJANGO_PRIVATE_MEDIA_ROOT=/data/private_media
     - DJANGO_RESULT_MEDIA_ROOT=/results
-    - DJANGO_CELERY_BROKER_URL=amqp://guest@rabbit
     - DJANGO_DATABASE_URL=postgis://postgres@database/postgres
```
Comment (das-g) [inline compose-common.yml]: Should `rabbitmqbase` below be removed, too?

Comment (hixi) [inline compose-common.yml]: yes :-)

**Pull request merged**